src/components/About.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import '../styles/About.css';
 
```

Comment:
- [ ] **Optional:** The about page is not rendered properly on the live page, kindly check it out. 

![image](https://user-images.githubusercontent.com/27709832/217902183-fc5be4ca-66f7-4e25-978f-0692e63527f2.png)